src/mapnode.cpp

```diff
@@ -584,6 +584,7 @@ s8 MapNode::setLevel(const NodeDefManager *nodemgr, s16 level)
 	s8 rest = 0;
 	const ContentFeatures &f = nodemgr->get(*this);
 	if (f.param_type_2 == CPT2_FLOWINGLIQUID
+			|| f.param_type_2 == CPT2_COLORED_FLOWINGLIQUID
```

Comment:
And again. Frankly I'm not sure if this PR should fix this redundancy since this was there before, but it is made a wee bit more annoying.